assets/search/components/TopicEditor.jsx

```diff
@@ -127,6 +127,11 @@ class TopicEditor extends React.Component {
                 !get(topic, field) :
                 event.target.value;
 
+            if (field === 'is_global') {
```

Comment:
If the user is currently in `My Topics`, won't this clear the list of user folders for the list view? Similar scenario, if the user is in `Company Topics` and disables the  `Share with my Company` checkbox?

We might need to have both user and company topics loaded in the store, so the list and form can use the respective folder type

Reply:
yep you are right, will change that